l3kernel/l3fp-functions.dtx

```diff
@@ -98,7 +98,7 @@
   }
 \cs_new:Npn \@@_function_set_parsing_aux:NNn #1#2#3
   {
-    \cs_set:Npe \@@_tmp:w
+    \cs_set_nopar:Npe \@@_tmp:w
```

Comment:
Why `nopar` here? Other than for vars and for `ltcmd` interfaces, we stick to `\cs_set:Npn`, etc.

Reply:
You're right, I should do in the reverse direction --- dropping `nopar` in `l3fp-symbolic.dtx`.